### ISSUE TITLE
would lambdas be supported?

### ISSUE THREAD

**Issue by hoebat** [athos/jise#4]
It'd be great to have direct lambda support instead of using the clojure.lang.IFn interface.

Is that on the roadmap and how difficult would it be to get working?

**Comment by athos** [comment]
Yeah, it's on the project's roadmap though there are still lots of things to do to get there. At least, I have in mind:

- Nested class support
- Generics support

Also, both probably require some more internal refinements, especially for internal type representation. I don't have any concrete ideas on these right now.
